client/src/pages/LoginPage.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import Box from "@material-ui/core/Box";
+
+import { LoginSocialFacebook } from "reactjs-social-login";
+import { FacebookLoginButton } from "react-social-login-buttons";
+
+import { userAuth } from "../redux/actions/userActions";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    textAlign: "center",
+    verticalAlign: "middle",
+    padding: "5%",
+  },
+  loginButton: {
+    justifyContent: "center",
+    maxWidth: "25%",
+  },
+}));
+
+const LoginPage = () => {
+  const dispatch = useDispatch();
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.root}>
+      <LoginSocialFacebook
+        className={classes.loginButton}
+        appId={process.env.REACT_APP_FACEBOOK_APP_ID}
```

Comment:
We probably don't want to use `env` variables on the client side since they are going to be public anyway. Just make this a const variable, otherwise this might add hassle to Heroku for setting up env.